app.js

```diff
@@ -14,16 +14,21 @@ app.get("/api/topics", getAllTopics)
 app.get("/api/articles", getAllArticles)
 app.get("/api/articles/:article_id", getArticleById)
 app.get("/api/articles/:article_id/comments", getCommentsByArticleId )
+app.post("/api/articles/:article_id/comments", postCommentByArticleId )
+
 
 app.get("*",(req,res,next) => {
     res.status(404).send({ msg: "Route does not exist" })
 })
 
 
 app.use((err, req, res, next) => {
-    if (err.code === "22P02") {
+    if ( ["22P02","23502"].includes(err.code) ) {
         res.status(400).send({msg: "Bad request!"});
-    } else {
+    } 
+    if(err.code === "23503"){
+        res.status(404).send({msg: "Not found"});
+    }else {
         next(err);
     }
```

Comment:
I would suggest creating an error handlers file at this point to keep your app from becoming saturated with error handlers :)